plugins/database/quota/src/main/java/org/apache/cloudstack/api/command/QuotaTariffCreateCmd.java

```diff
@@ -66,6 +66,9 @@ public class QuotaTariffCreateCmd extends BaseCmd {
             + " endDate=2009-06-03.")
     private Date endDate;
 
+    @Parameter(name = ApiConstants.POSITION, type = CommandType.INTEGER, description = "Position in the execution sequence for tariffs of the same type", since = "4.19.0.0")
```

Comment:
```suggestion
    @Parameter(name = ApiConstants.POSITION, type = CommandType.INTEGER, description = "Position in the execution sequence for tariffs of the same type", since = "4.20.0.0")
```